3-platform/shopping-cart/domain/src/main/java/com/_6core/platform/shopping/cart/domain/mapper/Boolean2ValidationResponseMapper.java

```diff
@@ -0,0 +1,15 @@
+package com._6core.platform.shopping.cart.domain.mapper;
+
+import com._6core.platform.shopping.cart.domain.dto.internal.ValidationResponse;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.factory.Mappers;
+
+@Mapper
+public interface Boolean2ValidationResponseMapper {
+  Boolean2ValidationResponseMapper INCTANCE =
```

Comment:
```suggestion
  Boolean2ValidationResponseMapper INSTANCE =
```
Change the name of the field in the Mappers to INSTANCE